resources/stylesheets/main.css

```diff
@@ -0,0 +1,344 @@
+    display: block; }
+    .nav-links ul li a {
+      font-size: 22px;
+      text-transform: uppercase;
+      padding: 5px 0;
+      font-weight: 600;
+      color: #70676a;
+      transition: 0.5s; }
+      .nav-links ul li a:hover, .nav-links ul li a:focus {
+        color: #bb2855; }
+
+/*** form css ***/
+.fieldset {
+  position: relative;
+  margin-bottom: 30px; }
+  .fieldset:last-child {
+    margin-bottom: 0px;
+    padding-top: 0; }
+  .fieldset input {
+    font-size: 18px;
+    padding: 10px 10px 10px 0px;
+    display: block;
+    width: 100%;
+    border: none;
+    border-bottom: 1px solid #acacac;
+    font-weight: 600; }
+    .fieldset input:focus {
+      outline: none; }
+    .fieldset input:focus ~ label, .fieldset input:valid ~ label {
+      top: -12px;
+      font-size: 12px;
+      color: #2fb9c6; }
+    .fieldset input:focus ~ .border_bottom:before, .fieldset input:focus ~ .border_bottom:after, .fieldset input:valid ~ .border_bottom:before, .fieldset input:valid ~ .border_bottom:after {
+      width: 50%; }
+  .fieldset label {
+    color: #acacac;
+    font-size: 14px;
+    font-weight: normal;
+    position: absolute;
+    pointer-events: none;
+    left: 0px;
+    top: 10px;
+    -o-transition: 0.2s ease all;
+    transition: 0.2s ease all;
+    -moz-transition: 0.2s ease all;
+    -webkit-transition: 0.2s ease all;
+    font-weight: 600;
+    font-size: 14px; }
+  .fieldset .border_bottom {
+    position: relative;
+    display: block;
+    width: 100%; }
+    .fieldset .border_bottom:before, .fieldset .border_bottom:after {
+      content: "";
+      height: 1px;
+      width: 0;
+      bottom: 0px;
+      position: absolute;
+      background: #2fb9c6;
+      -o-transition: 0.2s ease all;
+      transition: 0.2s ease all;
+      -moz-transition: 0.2s ease all;
+      -webkit-transition: 0.2s ease all; }
+    .fieldset .border_bottom:before {
+      left: 50%; }
+    .fieldset .border_bottom:after {
+      right: 50%; }
+
+.header__nav #nav-input:checked ~ label .hamburger {
+  width: 30px; }
+  .header__nav #nav-input:checked ~ label .hamburger__line1 {
+    width: 0;
+    transition-delay: 0.3s; }
+  .header__nav #nav-input:checked ~ label .hamburger__line2 {
+    width: 0;
+    transition-delay: 0.5s; }
+  .header__nav #nav-input:checked ~ label .hamburger__line3 {
+    width: 0;
+    transition-delay: 0.6s; }
+  .header__nav #nav-input:checked ~ label .hamburger__cross {
+    width: 25px;
+    transition-delay: 0.8s; }
+    .header__nav #nav-input:checked ~ label .hamburger__cross:after {
+      width: 25px;
+      transition-delay: 1.1s; }
+
+/* footer */
+footer {
+  background-color: #eaeaea;
+  padding: 16px 0;
+  text-align: center; }
+  footer p {
+    font-size: 16px; }
+
+/* footer */
+@media only screen and (min-width: 480px) {
+  .btn {
+    padding: 13px 17px;
+    font-size: 16px; } }
+
+@media only screen and (min-width: 768px) {
+  .header__logo {
+    width: 190px; }
+  .header__nav {
```

Comment:
Why you did not use SCSS, that was mandatory requirement!

Reply:
Actually the folder structure is like Resources and in this folder i have added SCSS and STYLESHEETS folder, where i am compiling scss files into css.